g11n-ws/modules/md-restful-l10n-synch/src/main/java/com/vmware/l10n/synch/controller/TranslationCommitGitController.java

```diff
@@ -46,6 +46,7 @@ public Response notifyCommit(@RequestBody SynchFile2GitReq synchFile2GitReq) {
 	} catch (InterruptedException e) {
 		// TODO Auto-generated catch block
 		logger.error(e.getMessage(), e);
+		Thread.currentThread().interrupt();
```

Comment:
why it need to change l10n component's codes?

Reply:
because this PR can't separate multi-component API and sarnoclude Scan bugs. this part is the scan bugs